src/tests/cmocka/test_utils.c

```diff
@@ -915,12 +915,11 @@ struct name_init_test_ctx {
 };
 
 #define GLOBAL_FULL_NAME_FORMAT "%1$s@%2$s"
-#define GLOBAL_RE_EXPRESSION "(?P<name>[^@]+)@?(?P<domain>[^@]*$)"
-
-#define TEST_DOMAIN_NAME "test.dom"
+#define TEST_DOMAIN_NAME_LDAP "test.dom"
+#define TEST_DOMAIN_NAME_IPA "test.ipa"
+#define TEST_DOMAIN_NAMES TEST_DOMAIN_NAME_LDAP "," TEST_DOMAIN_NAME_IPA
 #define DOMAIN_FULL_NAME_FORMAT "%3$s\\%1$s"
-#define DOMAIN_RE_EXPRESSION "(((?P<domain>[^\\\\]+)\\\\(?P<name>.+$))|" \
```

Comment:
Hi,

why this change (removal of '(((?P<domain>[^\\\\]+)\\\\(?P<name>.+$))') was required here?

Reply:
Originally this was a copy of IPA/AD default. I wanted to change it for two reasons

- clearly show that this is not the default value
- keep the test `test_sss_names_init` checking that we are reading value from config, not the default

Reply:
Thanks.